redash/serializers/__init__.py

```diff
@@ -272,33 +271,46 @@ def serialize(self):
 
 
 def serialize_job(job):
+    # TODO: this is mapping to the old Job class statuses. Need to update the client side and remove this
```

Comment:
> Need to update the client side and remove this

Is this meaning a follow up PR/commit, or was it more a note to yourself while going through it, or something else? :smile:

Reply:
No worries. :smile: